server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/main/java/org/apache/james/webadmin/vault/routes/DeletedMessagesVaultRoutes.java

```diff
@@ -149,11 +158,19 @@ public void define(Service service) {
             paramType = "query",
             example = "?action=restore",
             value = "Compulsory. Needs to be a valid action represent for an operation to perform on the Deleted Message Vault, " +
-                "valid action should be in the list (restore)")
+                "valid action should be in the list (restore, export)"),
+        @ApiImplicitParam(
+            dataType = "String",
+            name = "exportTo",
+            paramType = "query",
+            example = "?exportTo=user@james.org",
+            value = "Compulsory if action is export. Needs to be a valid mail address to represent for the destination " +
+                "where deleted messages content is export to")
```

Comment:
Needs to be a valid mail address. The content of the vault matching the query will be sent to that address.